server/services/propertyVideo.service.js

```diff
@@ -60,6 +61,14 @@ export const updateVideo = async (updatedVideo) => {
     throw new ErrorHandler(httpStatus.FORBIDDEN, 'You are not permitted to perform this action');
   }
 
+  let { status } = video;
```

Comment:
Even though this will no longer be valid as we are removing approved, you could have written the lines together as one.

```suggestion
  let { approved, status } = video;
```